.github/workflows/main.yml

```diff
@@ -130,11 +124,11 @@ jobs:
       - name: Setup Python
         uses: actions/setup-python@v4
         with:
-          python-version: 3.11-dev
+          python-version: 3.x
```

Comment:
Hmm… looking at the upstream project, they did the same thing.

Reply:
Yeah, everything in this PR came directly from upstream. I didn't look into the changes too closely. I know that skeleton is used by hundreds or maybe thousands of other projects, so presumably if there were a mistake here, it should have been caught already - and if not, there will be another update in the upstream project to fix it and we can just merge that in.

Anyway, since you mentioned it, I got curious and went to look up [what the setup-python action has to say about it](https://github.com/actions/setup-python/blob/main/docs/advanced-usage.md#using-the-python-version-input). Apparently `3.x` is a [version specification borrowed from npm](https://github.com/npm/node-semver#x-ranges-12x-1x-12-); it means the same as `>=3, <4`.